js/main.js

```diff
@@ -143,6 +155,18 @@ function viewSwapDataViews(dataView) {
     main.removeChild(document.querySelector('div[data-view="profile"]'));
     main.appendChild(profileDOM(data));
   }
+
+  if (dataView === 'edit-profile') {
+
+    document.forms[0].avatarUrl.value = data.profile.avatarUrl;
+    document.forms[0].username.value = data.profile.username;
+    document.forms[0].fullName.value = data.profile.fullName;
+    document.forms[0].location.value = data.profile.location;
+    document.forms[0].bio.value = data.profile.bio;
+
+    imagePlaceholder.setAttribute('src', data.profile.avatarUrl);
```

Comment:
Isn't this overwriting your placeholder image? I assume you want to overwrite the actual profile image `src`